src/oscar/templates/oscar/customer/emails/base.html

```diff
@@ -1,6 +1,296 @@
+              text-align: left;
+            }
+
+            .clear {
+              clear: both;
+            }
+
+            .alert {
+              font-size: 16px;
+              color: #fff;
+              font-weight: 500;
+              padding: 20px;
+              text-align: center;
+              border-radius: 3px 3px 0 0;
+            }
+            .alert a {
+              color: #fff;
+              text-decoration: none;
+              font-weight: 500;
+              font-size: 16px;
+            }
+            .alert.alert-warning {
+              background-color: #FF9F00;
+            }
+            .alert.alert-bad {
+              background-color: #D0021B;
+            }
+            .alert.alert-good {
+              background-color: #68B90F;
+            }
+
+            .invoice {
+              margin: 40px auto;
+              text-align: left;
+              width: 80%;
+            }
+            .order {
+              text-align: left;
+              width: 100%;
+            }
+            .invoice td, .order td {
+              padding: 5px 0;
+            }
+            .invoice .invoice-items, .order .order-items {
+              width: 100%;
+            }
+            .invoice .invoice-items td, .order .order-items td {
+              border-top: #eee 1px solid;
+            }
+            .invoice .invoice-items .total td, .order .order-items .total td {
+              border-top: 2px solid #333;
+              border-bottom: 2px solid #333;
+              font-weight: 700;
+            }
+
+            @media only screen and (max-width: 640px) {
+              body {
+                padding: 0 !important;
+              }
+
+              h1, h2, h3, h4 {
+                font-weight: 800 !important;
+                margin: 20px 0 5px !important;
+              }
+
+              h1 {
+                font-size: 22px !important;
+              }
+
+              h2 {
+                font-size: 18px !important;
+              }
+
+              h3 {
+                font-size: 16px !important;
+              }
+
+              .container {
+                padding: 0 !important;
+                width: 100% !important;
+              }
+
+              .content {
+                padding: 0 !important;
+              }
+
+              .content-wrap {
+                padding: 10px !important;
+              }
+
+              .invoice {
+                width: 100% !important;
+              }
+
+              .order {
+                width: 100% !important;
+              }
+            }
+
+        </style>
+    </head>
+
+    <body itemscope itemtype="http://schema.org/EmailMessage">
+
+    <table class="body-wrap">
+        <tr>
+            <td></td>
+            <td class="container" width="600">
+                <div class="content">
+                    <table class="main" width="100%" cellpadding="0" cellspacing="0" itemprop="action" itemscope itemtype="http://schema.org/ConfirmAction">
```

Comment:
Mailgun inlines all the styles (font, size etc) - see https://github.com/mailgun/transactional-email-templates/blob/master/templates/inlined/alert.html#L60 . Did you deliberately remove those?

Reply:
No, just used generic template version without inlines.